src/lightning/pytorch/utilities/imports.py

```diff
@@ -32,6 +32,7 @@
 _RICH_AVAILABLE = package_available("rich") and compare_version("rich", operator.ge, "10.2.2")
 _TORCH_QUANTIZE_AVAILABLE = bool([eg for eg in torch.backends.quantized.supported_engines if eg != "none"])
 _TORCHVISION_AVAILABLE = RequirementCache("torchvision")
+_LIGHTNING_COLOSSALAI_AVAILABLE = package_available("lightning_colossalai")
```

Comment:
RequirementCache plz?

Reply:
It wasn't working when I was tinkering with the package. Let's try again.